docs/user-guide/platform-linux.md

```diff
@@ -78,6 +77,14 @@ cd pegasus-frontend-git
 makepkg -si
 ```
 
+!!! info "Steam Deck"
+
+Currently, Steam OS is missing 2 required plugins for gstreamer in order for video playback to work in the standalone version. In order to install them, you must temporarily disable Read Only mode, install them and then re-enable Read Only mode as folows:
```

Comment:
- There's a typo in `folows`
- Let's write gstreamer as GStreamer
- Style: Avoid repeating sections like  `In order...`; instead, you can write `To install them, you must temporarily disable Read Only mode, install the plugins, and...`

Reply:
Yes, there is.

Reply:
Please don't forget about the second and third bullet points either :)

Reply:
Complete and resubmitted